pyocd/debug/rtt.py

```diff
@@ -421,10 +421,7 @@ def __init__(self, target: SoCTarget, address: int = None,
             self._cb_search_size_words = 0
         else:
             self._cb_search_size_words = size // 4
```

Comment:
`self._cb_search_size_words` is set to the size in words here, but it's used as a byte count for `.read_memory_block8()` in the search routine.

So, please remove the divide by 4 and rename the attribute to `_cb_search_size_bytes`.